packages/env-utils/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
nit: I would prefer to disable this rule in that one file only

Reply:
very good point, thanks 3048f25d35f3ce7be06b4c9d4afeb3e21bc5bc65